warehouse/query-core/src/test/java/datawave/query/testframework/AccumuloSetupHelper.java

```diff
@@ -89,14 +90,28 @@ public AccumuloSetupHelper(final Collection<DataTypeHadoopConfig> types, FileTyp
      */
     public Connector loadTables(final Logger parentLog) throws AccumuloException, AccumuloSecurityException, IOException, InterruptedException,
                     TableExistsException, TableNotFoundException, URISyntaxException {
+        return loadTables(parentLog, RebuildingScannerTestHelper.TEARDOWN.EVERY_OTHER);
```

Comment:
All of the exiting tests will call this loadTables method. Perhaps I am missing but I am not sure why the teardown would be EVERY_OTHER. I am assuming this is the current behavior.

Reply:
I am using EVERY_OTHER instead of the RANDOM one to ensure we have deterministic test cases.  I could to ALWAYS, but I need to test both the teardown cases and the non-teardown cases.  Hence tearing down the iterator stack every other time seemed like a good alternative.  In real life, the tear downs would happen at seemingly random times.